packages/@aws-cdk/aws-iam/test/merge-statements.test.ts

```diff
@@ -7,23 +7,6 @@ const principal = new iam.ArnPrincipal(PRINCIPAL_ARN);
 const PRINCIPAL_ARN2 = 'arn:aws:iam::111111111:role/role-name';
 const principal2 = new iam.ArnPrincipal(PRINCIPAL_ARN2);
```

Comment:
All of these tests have principals - perhaps it's worth adding a test to validate what we expect to happen for identity-based policies (where no principals are specified)?

Reply:
Good call